SCons/Builder.py

```diff
@@ -353,13 +353,21 @@ def __call__(self, target, source, env):
 
         return (target, source)
 
-
     def __eq__(self, other):
         return self.var == other.var
 
     def __lt__(self, other):
         return self.var < other.var
 
+    def __le__(self, other):
```

Comment:
These shouldn't be showing as added in this PR, as they're already in master? A bunch of other changes the same, too.